platform-stack/register_mqtt_sensor.sh

```diff
@@ -0,0 +1,381 @@
+#Provision a service group:
+sudo curl -iX POST \
+  'http://localhost:4041/iot/services' \
+  -H 'Content-Type: application/json' \
+  -H 'fiware-service: openiot' \
+  -H 'fiware-servicepath: /' \
+  -d '{
+ "services": [
+   {
+     "apikey":      "4jggokgpepnvsb2uv4s40d59ov",
+     "entity_type": "TrafficFlowObserved",
+     "resource":    ""
+   },
+   {
+     "apikey":      "4jkkokgpepnvsb0sd7q21t53tu",
+     "entity_type": "Lights",
+     "resource":    ""
+   },
+   {
+     "apikey":      "parkingApi",
+     "entity_type": "Parking",
+     "resource":    ""
+   },
+   {
+     "apikey":      "EvAPI",
+     "entity_type": "Ev-Charging",
+     "resource":    ""
+   }
+ ]
+}'
+
+#register mqtt device
+sudo curl -iX POST \
+  'http://localhost:4041/iot/devices' \
+  -H 'Content-Type: application/json' \
+  -H 'fiware-service: openiot' \
+  -H 'fiware-servicepath: /' \
+  -d '{
+    "devices": [
+      {
+        "device_id":   "traffic001",
+        "entity_name": "urn:ngsi-ld:TrafficFlowObserved:001",
+        "entity_type": "TrafficFlowObserved",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "car_counter", "name": "count", "type": "Integer" }
```

Comment:
attributes and commands you can put them directly in the service description and leave empty the device (this reduces duplications)

the name of the attribute here should be `intensity`